### ISSUE TITLE
Images must have alternate text

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/Adapt-accessibilityfixes#99]
Issue: Under "Being a Visible Minority in the Workplace", the pictures lack alternative text. Hovering with a mouse, the screen reader is silent. Solution: Each picture should have an alternative text like "Person 1", "Person 2", etc..

**Comment by MeD-Owner** [comment]
Content issue

**Comment by MeD-Owner** [comment]
Good suggestion!

**Comment by MeD-Owner** [comment]
Fixed! 20210602
